engine/src/main/scala/cromwell/engine/workflow/lifecycle/deletion/DeleteWorkflowFilesActor.scala

```diff
@@ -75,7 +76,10 @@ class DeleteWorkflowFilesActor(rootWorkflowId: RootWorkflowId,
         TryUtil.sequence(intermediateFiles.toSeq.map(gcsCommandBuilder.deleteCommand(_, swallowIoExceptions = false)))
       deleteCommandsTry match {
         case Success(deleteCommands) =>
-          deleteCommands foreach sendIoCommand
+          deleteCommands foreach { cmd =>
+            sendIoCommand(cmd)
+            log.info(s"Calling '$cmd' to delete intermediary file for root workflow $rootWorkflowId.")
```

Comment:
I think it would be better to just print the file name instead of the entire IoCommand object. This might work:
```
log.info(s"Calling '${cmd.file.pathAsString}' to delete intermediary file for root workflow $rootWorkflowId.")
```

Reply:
I modified what I'm logging to only include the file path.